packages/@aws-cdk/aws-cognito/package.json

```diff
@@ -102,7 +102,10 @@
     ]
   },
   "stability": "experimental",
-  "maturity": "experimental",
+  "featureStability": [
```

Comment:
How does one handle the value of stability here that is consumed by JSII and applies this to the docstring of all APIs in the library, when one of the features becomes 'stable'?